engines/e_ossltest.c

```diff
@@ -262,6 +264,18 @@ int ossltest_aes128_gcm_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
                                const unsigned char *in, size_t inl);
 static int ossltest_aes128_gcm_ctrl(EVP_CIPHER_CTX *ctx, int type, int arg,
                                     void *ptr);
+int ossltest_aes128_cbc_hmac_sha1_init_key(EVP_CIPHER_CTX *ctx,
+                                           const unsigned char *key,
+                                           const unsigned char *iv, int enc);
+int ossltest_aes128_cbc_hmac_sha1_cipher(EVP_CIPHER_CTX *ctx,
+                                         unsigned char *out,
+                                         const unsigned char *in, size_t inl);
```

Comment:
Should these functions be static as well?

Reply:
yeah, maybe when I'm already there, I should also do the same with AES-128-CBC and AES-128-GCM ciphers.